packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -61,13 +62,22 @@ export default new Bundler({
 
             // Mark bundle as an entry, and set explicit file path from target if the dependency has one
             bundle.isEntry = !!dep.isEntry;
-            if (dep.target && dep.target.distPath) {
-              bundle.filePath = dep.target.distPath;
+            let target = dep.target;
+            if (
+              target &&
+              target.distPath &&
+              target.distPathType === bundle.type
```

Comment:
Previously all entries would have the name `distPath` (css was written to `index.js` for example). I added `distPathType` to identify the specific entry this was intended for. Not sure about the case with multiple entries of this type.